lib/util/concerns/hud_lists_2024.rb

```diff
@@ -709,10 +707,9 @@ def funding_sources
         43 => 'HUD: CoC - Youth Homeless Demonstration Program (YHDP)',
         44 => 'HUD: CoC - Joint Component TH/RRH',
         45 => 'VA: Grant Per Diem - Case Management/Housing Retention',
-        46 => 'Local or Other Funding Source',
+        46 => 'Local or Other Funding Source (Please Specify)',
         47 => 'HUD: ESG - CV',
         48 => 'HUD: HOPWA - CV',
-        49 => 'HUD: CoC - Joint Component RRH/PSH [Deprecated]',
```

Comment:
I'm pretty sure this is fine, i.e. won't have any impact in hmis. I just want to flag that we had a convo around intentionally keeping deprecated options around, so that they show up in the HMIS as their real value, rather than just "invalid value". @eanders  I think that was still the right decision. Will there be a way to do that going forward?